tests/oldcheck/tests.yaml

```diff
@@ -0,0 +1,1341 @@
+    tuples:
+      - object: document:1
+        relation: writer
+        user: aardvark
+      - object: document:1
+        relation: editor
+        user: aardvark
+      - object: document:2
+        relation: writer
+        user: badger
+      - object: document:2
+        relation: owner
+        user: badger
+      - object: document:3
+        relation: writer
+        user: cheetah
+    assertions:
+      - tuple:
+          object: document:1
+          relation: viewer
+          user: aardvark
+        expectation: false
+      - tuple:
+          object: document:2
+          relation: viewer
+          user: badger
+        expectation: false
+      - tuple:
+          object: document:3
+          relation: viewer
+          user: cheetah
+        expectation: true
+
+  - name: ExclusionAndExclusionInSubtract
+    model: |
+      type document
+        relations
+          define writer as self
+          define editor as self
+          define owner as self
+          define viewer as writer but not (editor but not owner)
+    tuples:
+      - object: document:1
+        relation: writer
+        user: aardvark
+      - object: document:1
+        relation: editor
+        user: aardvark
+      - object: document:1
+        relation: owner
+        user: aardvark
+      - object: document:2
+        relation: writer
+        user: badger
+      - object: document:2
+        relation: editor
+        user: badger
+      - object: document:3
+        relation: writer
+        user: cheetah
+      - object: document:3
+        relation: owner
+        user: cheetah
+    assertions:
+      - tuple:
+          object: document:1
+          relation: viewer
+          user: aardvark
+        expectation: true
+      - tuple:
+          object: document:2
+          relation: viewer
+          user: badger
+        expectation: false
+      - tuple:
+          object: document:3
+          relation: viewer
+          user: cheetah
+        expectation: true
+
+  - name: userset_as_user
+    model: |
+      type group
+        relations
+          define member as self
+      
+      type document
+        relations
+          define viewer as self
+    tuples:
+      - object: document:1
+        relation: viewer
+        user: group:x#member
+    assertions:
+      - tuple:
+          object: document:1
+          relation: viewer
+          user: group:x#member
+        expectation: true
+
+  - name: wildcard_direct
+    model: |
+      type document
+        relations
+          define viewer as self
+    tuples:
+      - object: document:public
+        relation: viewer
+        user: "*"
+    assertions:
+      - tuple:
+          object: document:public
+          relation: viewer
+          user: aardvark
+        expectation: true
+
+  - name: typed_wildcard_string_is_treated_as_a_literal
+    model: |
+      type document
+        relations
+          define viewer as self
+    tuples:
+      - object: document:public
```

Comment:
This isn't a public document, so this is misleading. `user:*` is the literal user_id in 1.0 models, so this isn't a public document.

Reply:
Fixed. Thanks!